### DISCUSSION TITLE
Extensión informe

### DISCUSSION THREAD

**Discussion by sbarriag** [IIC2133-PUC/2024-2#114] [Tarea 1] [answered]
Hola, queria preguntar si existe la posibilidad de extender un poco el límite de extensión del informe, creo que es dificil justificar lo que nos piden en solo 2 páginas, entiendo que tampoco nos piden una documentación gigante ni nada por el estilo, pero como nos piden explicar algoritmos, demostrar su complejidad, explicar como funciona nuestro código encuentro que 2 páginas no pueden contener todo eso.

Saludos y muchas gracias por la consideración

**Comment by sebacarrascop** [comment, marked as answer]
Hola!, dado que el informe se debe revisar manualmente, extender el largo implica mucho más trabajo de corrección, por lo que no es viable. Como mencionas, no se pide una documentación gigante ni nada por el estilo, lo que se busca que se entregue en el informe se puede entregar perfectamente en el largo límite.
Saludos.

**Reply by AntonioEscobar01** [reply]
Pero si en el mismo teamplate una función en seudocódigo usa media plana y cada parte probablemente tenga más de una sola función, con esa extensión probablemente el informe no logre explicar todo lo que hicimos

**Reply by sebacarrascop** [reply]
hola!, no se pide poner un pseudo código de todas las funciones que hicieron ni explicar exactamente todo lo que hicieron en la tarea, varios alumnos ya han subido el informe en el largo pedido y entregando lo solicitado sin problemas, saludos.
